src/components/App.js

```diff
@@ -19,9 +22,12 @@ const App = () => {
         <div>
           <Route path="/" exact component={HomeContainer} />
           <Route path="/about" component={AboutContainer} />
-          <Route path="/causes" component={CausesContainer} />
+          <Route path="/causes" exact component={CausesContainer} />
```

Comment:
@javpet I needed to add the `exact` attribute to the "index" routes (i.e. `/causes`, `/performances`, and `/artists`), otherwise, because the routes for the individual containers have the same root phrase within their own route (e.g. `/causes/5`), it would render both the index and the individual containers when navigating to the individual route.